src/Pages/BeerGoggles/BeerGoggles.css

```diff
@@ -0,0 +1,54 @@
+.beerGooglesContainer {
+    max-width: 1200px;
+    margin: auto;
+}
+
+.beerGooglesIntroContainer {
+    display: flex;
+    padding-bottom: 30px;
+}
+
+.beerGooglesDescription {
```

Comment:
This styling as well as below feels like it is generic enough that it will be used across multiple Project pages so maybe it should be a more generic class in the main css?